types/openai.ts

```diff
@@ -9,9 +9,12 @@ export interface OpenAIModel {
 
 export enum OpenAIModelID {
   GPT_3_5 = 'gpt-3.5-turbo',
-  GPT_3_5_AZ = 'gpt-35-turbo',
-  GPT_4 = 'gpt-4',
-  GPT_4_32K = 'gpt-4-32k',
+  GPT_3_5_AZ = 'gpt-35-turbo-1106',
+  GPT_3_5_16k = 'gpt-35-turbo-16k-0613',
+  GPT_4 = 'gpt-4-0613',
+  GPT_4_32K = 'gpt-4-32k-0613',
+  GPT_4_TURBO = 'gpt-4-1106-Preview',
```

Comment:
Just updated available models while I was already here. I've tested all the ones that are available as well as gpt-4-vision-preview locally. Everything seems to be working.